src/share/vm/classfile/javaClasses.cpp

```diff
@@ -977,6 +978,11 @@ oop java_lang_Thread::name(oop java_thread) {
 }
 
 
+oop java_lang_Thread::resourceContainer(oop java_thread) {
+  return java_thread->obj_field(_resourceContainer_offset);
+}
+
+
```

Comment:
please remove extra blank line

Reply:
seems no need to remove(2 blank lines between functions)